docs/notebooks/model-levels.ipynb

```diff
@@ -0,0 +1,2052 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Working with Model Level Met\n",
+    "\n",
+    "Meteorology providers may choose to compute their data on different vertical levels. For example, the [ECMWF IFS product](https://confluence.ecmwf.int/display/UDOC/L137+model+level+definitions) is computed on  data on 137 model levels. The pressure of a given model level is not constant, but varies with time and horizontal position. The pressure at a given point can be computed [following ECMWF guidelines](https://confluence.ecmwf.int/display/CKB/ERA5%3A+compute+pressure+and+geopotential+on+model+levels%2C+geopotential+height+and+geometric+height). This notebook uses the `metview` library to convert from ECMWF model levels to pressure levels.\n",
```

Comment:
I'd put the link on the text [137 model levels](https://confluence.ecmwf.int/display/UDOC/L137+model+level+definitions)

Reply:
4377b6e67178079f6262753d7226d695e0829883